swap/src/protocol/bob/event_loop.rs

```diff
@@ -270,6 +271,7 @@ impl EventLoopHandle {
         match response {
             Response::Xmr(xmr) => Ok(xmr),
             Response::Error(error) => {
+                let error: bob::spot_price::Error = error.into();
```

Comment:
I wonder how we could organize our modules such that we don't need to over-qualify the type here given that we already import another module named `spot_price` and we are in a module scoped to `bob` so `bob::spot_price` is pretty redundant.

Reply:
One thing that came to my mind is, that this network related stuff should not be under `protocol` anyway!
I think we should introduce `alice` and `bob` in the network module instead and then decide if something is global, alice or bob on the network level, independent of the protocol.

Will see to give this more thought and tackle this in follow-up(s)